src/System.Runtime.Intrinsics.Experimental/ref/System.Runtime.Intrinsics.Experimental.cs

```diff
@@ -7,6 +7,7 @@
 
 namespace System.Runtime.Intrinsics.Arm.Arm64
 {
+    [System.CLSCompliantAttribute(false)]
```

Comment:
I think you had a discussion about this where you explained the reasoning, right?

Reply:
For ARM64, I think the initial push just marked them all as `CLSCompliant(false)` to avoid warnings. We should probably update that to correctly annotate individual members once it is no longer experimental.